src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -71,7 +102,7 @@ export class GKEClusterSettingsComponent
   ngOnInit(): void {
     this._initForm();
     this._initSubscriptions();
-    this._getGKEZones();
+    !this.isDialogView() && this._getGKEZones();
```

Comment:
Avoid single line shortcuts
```suggestion
 if(!this.isDialogView()) {
      this._getGKEZones();
    }
```